app/src/main/java/com/example/kuba10/mypokemonplaces/AddPlaceFragment/AddPlaceFragment.java

```diff
@@ -1,169 +0,0 @@
-    @BindView(R.id.title_input)
-    TextView titleField;
-    @BindView(R.id.add_button)
-    Button addButt;
-    @BindView(R.id.cancel_button)
-    Button cancelButt;
-
-    public static AddPlaceFragment newInstance(LatLng position) {
-        AddPlaceFragment fragment = new AddPlaceFragment();
-        Bundle args = new Bundle();
-        args.putParcelable(Constants.POSITION, position);
-        fragment.setArguments(args);
-        return fragment;
-    }
-
-
-    @Override
-    public void onAttach(Context context) {
-        super.onAttach(context);
-        fragmentListener = (FragmentListener) context;
-        pokemonGo_data_list = fragmentListener.getPokemonList();
-    }
-    @Override
-    public View onCreateView(LayoutInflater inflater, ViewGroup container,
-                             Bundle savedInstanceState) {
-        View view = inflater.inflate(R.layout.fragment_add_place, container, false);
-
-        ButterKnife.bind(this, view);
-
-        addButt.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View view) {
-
-                LatLng position = getArguments().getParcelable(Constants.POSITION);
-                double lat = position.latitude;
-                double lng = position.longitude;
-
-                PokePlace place = new PokePlace();
-
-                if (titleField.getText().toString().equals("")) {
-                    fragmentListener.showSnackbar(getString(R.string.placeMustHaveTitle));
-                } else {
-
-                    place.setTitle(titleField.getText().toString());
-                    place.setTitle(titleField.getText().toString());
-                    place.setDesctription(descriptionField.getText().toString());
-                    place.setLong(lng);
-                    place.setLat(lat);
-                    place.setListPosition(Constants.INDEX_NOT_SET);
-                    place.setFavourite(0);
-
-                    setSelectedPokemonId(place);
-                    setPlaceId(place);
-
-                    fragmentListener.savePlace(place);
-
-                    dismiss();
-
-                }
-
-
-            }
-        });
-        cancelButt.setOnClickListener(new View.OnClickListener()
-        {
-            @Override
-            public void onClick(View view) {
-                dismiss();
-            }
-        });
-        pokemonImageView.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View view) {
-                fragmentListener.openFragment(ChooseFragment.newInstance(pokemonGo_data_list));
-
-            }
-        });
-        return view;
-    }
-
-    private void setPlaceId(PokePlace place) {
-        Date date = new Date();
-        Long id = date.getTime();
-        place.setGlobalID(id);
-    }
-
-    private void setSelectedPokemonId(PokePlace place) {
-        if(selectedPokemon == null) {
-            place.setPokemonId(-777);
```

Comment:
-777 do stałych. Jeśli nie ma powodu dla tej wartości, daj `Integer.MAX` albo coś tego typu

Reply:
Wyrzuciłem jako stałą w constants - to akurat jest w kilku miejscach projektu

Reply:
Jeśli dotyczy pokemona to daj ją do `Pokemon` tylko zrób ją po prostu publiczną. Uwierz, od razu widać o co chodzi :smiley: . Wtedy na przykład nie musisz jej nazywać `POKEMON_ID` tylko `ID` bo wywołujesz ją z klasy Pokemon (bo pokemona dotyczy) czyli `Pokemon.ID`

Reply:
+ w Twoim przypadku to coś w stylu `Pokemon.INVALID_ID` albo podobne